package.json

```diff
@@ -18,7 +18,7 @@
     "canvas-prebuilt": "^1.6.0-canvas.5",
     "chalk": "^2.0.0",
     "chrome-launcher": "^0.6.0",
-    "chrome-remote-interface": "^0.24.4",
+    "chrome-remote-interface": "^0.24.5",
```

Comment:
Use `Upgrade:`  instead of `Chore:`, unless upgrading the dependency breaks something.